lib/commands/install.sh

```diff
@@ -41,14 +41,16 @@ install_local_tool_versions() {
   asdf_versions_path=$(find_tool_versions)
   if [ -f "${asdf_versions_path}" ]; then
     while IFS= read -r tool_line || [ -n "$tool_line" ]; do
-      IFS=' ' read -r -a tool_info <<< "$tool_line"
-      local tool_name
-      tool_name=$(echo "${tool_info[0]}" | xargs)
-      local tool_version
-      tool_version=$(echo "${tool_info[1]}" | xargs)
+      if ! (echo "$tool_line" | grep "^#.*" > /dev/null); then
```

Comment:
I think I would probably rather go with something like

```shell
tool_line="$(echo "$tool_line" | sed -r -e 's/(\#.*)//')"
```

and leave the decision of installing or not to the condition we already have: 

```
if ! [[ -z "$tool_name" || -z "$tool_version" ]];
```

This would allow inline comments:

```
python 3.5.2 2.7.15  # fallback to py2 if command not found in py3
nodejs 8.12.0  # anything above 8 should work
``` 

What do you think?

Reply:
Good idea. I've updated the code (and added unit tests) so comments are where ever they appear - on their own line, or at the end of a version line.